### ISSUE TITLE
PCM channel not working on exported builds

### ISSUE THREAD

**Issue by Weeppiko** [nesbox/TIC-80#2615]
Executables built with the `export` command have their PCM channel completely silent

**Comment by nesbox** [comment]
Yes, because we have old binaries for the export on the server, we need to set up our pipeline properly to update them on every build.
